src/overlays/actors/ovl_En_Mb/z_en_mb.c

```diff
@@ -7,10 +7,49 @@
 #include "z_en_mb.h"
 #include "objects/object_mb/object_mb.h"
 
+/*
+ * This actor can have three behaviors:
+ * - "Spear Guard" (variable -1): uses a spear, walks around home point, charges player if too close
+ * - "Club" (variable 0): uses a club, stands still and smashes its club on the ground when the player approaches
+ * - "Spear Patrol" (variable 0xPP00 PP=pathId): uses a spear, patrols following a path, charges
+ */
+
 #define FLAGS 0x00000015
 
 #define THIS ((EnMb*)thisx)
 
+typedef enum {
+    /* -1 */ ENMB_TYPE_SPEAR_GUARD = -1,
+    /*  0 */ ENMB_TYPE_CLUB,
+    /*  1 */ ENMB_TYPE_SPEAR_PATROL
+} EnMbType;
```

Comment:
Params enums usually go in the header so they can be accessed externally. Otherwise looks good!

Reply:
makes sense but I guess another enum would have to be created, as `ENMB_TYPE_SPEAR_PATROL == 1` is only used in code (it's set by code)
actors lists in room files use either `0x0000` for the Club Moblin or `0xPPFF` (`PP=pathId`) for the Spear Patrol Moblin
the lower byte can be anything as long as the whole variable isn't 0 or -1 so `ENMB_TYPE_SPEAR_PATROL == 1` would work, but apparently they used `0xFF`

so, do move it or just leave it and wait for another params pass

Reply:
moving it is fine. even if some entries are not "public", in general you will be accessing type externally